RealmSwift-swift2.0/Migration.swift

```diff
@@ -70,7 +70,9 @@ block automatically as needed.
 */
 @available(*, deprecated=1, message="Use Realm(configuration:error:)")
 public func setDefaultRealmSchemaVersion(schemaVersion: UInt64, migrationBlock: MigrationBlock) {
-    RLMRealmSetSchemaVersionForPath(schemaVersion, Realm.Configuration.defaultConfiguration.path, accessorMigrationBlock(migrationBlock))
+    RLMRealmSetSchemaVersionForPath(schemaVersion,
+        Realm.Configuration.defaultConfiguration.path,
+        accessorMigrationBlock(migrationBlock))
```

Comment:
since this PR is about style, might as well discuss it further here. my personal preference is to stay consistent within a function call or declaration: if placing one parameter on a newline, might as well do the same with all parameters.